### DISCUSSION TITLE
Question on reading zip archives

### DISCUSSION THREAD

**Discussion by Mafuyu0** [happypandax/happypandax#275] [Q&A] [answered]
I am curious: When the app is reading something stored on the server as a zip file, where does the server temporarily extract the zip? Is this stored in `download_cache_dir` or some other place?

I am running this one windows and I just want to make sure there are no temporary files getting leaked in some other temporary directory.

**Comment by twiddli** [comment, marked as answer]
With default settings, everything is self-contained and nothing gets generated/written outside of the HPX folder.
When viewing, images are extracted to folders `[HPX ROOT]/static/(pages|thumbnails)`, and are regularly trimmed according to `core.auto_pages_clean_size` and `core.auto_thumbs_clean_size`.
